### PR TITLE
Trying newest mixer

### PR DESCRIPTION
**Pull request by ldemailly**
```release-note
none
```

### CODE REVIEW

**Comment by ldemailly** [comment]
(made https://github.com/istio/istio/issues/642 for the bogus space issue)

File: install/kubernetes/istio-auth.yaml [hunk 1]
```diff
@@ -218,18 +207,12 @@ spec:
         - mountPath: /etc/certs
           name: istio-certs
           readOnly: true
-        - mountPath: /etc/istio/ingress-certs
```
Comment (kyessenov) [inline install/kubernetes/istio-auth.yaml]: we can no longer remove these since pilot PR got merged to require these

Comment (ldemailly) [reply install/kubernetes/istio-auth.yaml]: well the template needs to be fixed... any change (including the bot auto update) will erase these until the template is fixed

Comment (andraxylia) [reply install/kubernetes/istio-auth.yaml]: yes, you cannot remove the imgress secrets.

Comment (myidpt) [reply install/kubernetes/istio-auth.yaml]: Which template do we use to generate these files?

Comment (ldemailly) [reply install/kubernetes/istio-auth.yaml]: the one mentioned at the beginning of the file

Comment (ldemailly) [reply install/kubernetes/istio-auth.yaml]: https://github.com/istio/istio/blob/master/install/updateVersion.sh#L155

Comment (myidpt) [reply install/kubernetes/istio-auth.yaml]: I will fix it in a separate PR.

**Comment by ldemailly** [comment]
going to close this until the other fix is done then rebase/redo

**Comment by ldemailly** [comment]
PTAL

**Comment by ldemailly** [comment]
will wait for https://github.com/istio/mixer/pull/1203
(and/or the bots to catchup though for the bots we need https://github.com/istio/pilot/pull/1168 )

**Comment by ldemailly** [comment]
closing this one actually as the bot PR #663 is better and I already fixed the pilot sha in another one and there is an even newer mixer

**Pull request closed without merging**